docs/marionette.region.md

```diff
@@ -257,18 +261,22 @@ This can be useful in unit testing your views.
 Override the region's `attachHtml` method to change how the view is attached
 to the DOM. This method receives one parameter - the view to show.
 
-The default implementation of `attachHtml` is:
+The default implementation of `attachHtml` works a bit like the following:
 
 ```javascript
 var Mn = require('backbone.marionette');
 
 Mn.Region.prototype.attachHtml = function(view){
-  this.$el.empty().append(view.el);
+  if (this.isShowingAView()) {
```

Comment:
I'm not sure this is true..

`attachHtml` is passed a flag as the 2nd argument that's used so that the new view's el can replace the current region's el.. otherwise it just appends..  it doesn't do anything regarding removing a view, and by this point there's no chance that a region has a view in it.